File: k-kbk/3주차/1476.py
e, s, m = map(int, input().split())

count = 1
x, y, z = 1, 1, 1

while True:
    if e == x and s == y and m == z:
        break

    x += 1
    if x == 16:
        x = 1
    y += 1
    if y == 29:
        y = 1
    z += 1
    if z == 20:
        z = 1

    count += 1

print(count)

File: keemsebin/3주차/1476.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let input = [];

rl.on("line", function (line) {
  input.push(line);
}).on("close", function () {
  let [E, S, M] = input[0].split(" ").map(Number);
  let year = 1;
  while (true) {
    if (
      (year - E) % 15 === 0 &&
      (year - S) % 28 === 0 &&
      (year - M) % 19 === 0
    ) {
      console.log(year);
      process.exit();
    }
    year++;
  }
});
